src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke {

```

Comment:
Consider adding more classes for the `Ui` and `Command` code to apply OOP techniques.